pybamm/models/base_model.py

```diff
@@ -421,31 +421,63 @@ def input_parameters(self):
             self._input_parameters = self._find_symbols(pybamm.InputParameter)
         return self._input_parameters
 
-    def print_parameter_info(self):
-        """Returns parameters used in the model"""
-        self._parameter_info = ""
+    def get_parameter_info(self):
+        """
+        Extract the parameter information and returns it as a dictionary.
+        To get a list of all parameter-like objects without extra information,
+        use `model.parameters`.
```

Comment:
```suggestion
        Extracts the parameter information and returns it as a dictionary.
        To get a list of all parameter-like objects without extra information,
        use :py:attr:`model.parameters`.
```

this is because reST syntax will interpret single backticks (``) as italic-styled text, but we want it to be highlighted as code